examples/cg.py

```diff
@@ -271,8 +276,43 @@ def run_cg(
         dest="conv_threshold",
         help="convergence check threshold",
     )
+    parser.add_argument(
+        "--package",
+        dest="package",
+        type=str,
+        default="legate",
+        help="NumPy package to use (legate, numpy, or cupy)",
+    )
+    parser.add_argument(
+        "--cupy-allocator",
+        dest="cupy_allocator",
+        type=int,
+        default=0,
+        help="""cupy allocator to use (0-default memory pool,
+             1-turn off memory pool, 2-managed memory pool)""",
+    )
 
     args = parser.parse_args()
+
+    if args.package == "legate":
+        try:
+            import cunumeric as np
+        except ImportError:
+            import legate.numpy as np
```

Comment:
Just `import cunumeric as np`. Same applies to other files.

Reply:
Fixed. Thanks.

Reply:
This was a leftover to allow testing of pre-cunumeric code, but I guess we don't need that anymore.